playbooks/luna_demo_environment.yml

```diff
@@ -0,0 +1,79 @@
+# Manually verify your hosts has FQDN set correctly
+# check it's in /etc/hosts /etc/hostname and hostname returns it, /etc/hosts first line should be 192.168.73.1  $fqdn
+
+# TODO - add role for setting up the machine - autologin in gdm, perhaps background image etc, installation of virt-manager, vim, htop
+# TODO - refactor katello_provisioning so that it does not run installer, it should only seed stuff, installer should be customized from playbook using foreman_installer's role variables
+
```

Comment:
You can test this in a vm created automatically by forklift by adding the following:

```yaml
- hosts: localhost
  vars:
    forklift_name: centos7-luna-demo
    forklift_boxes:
      centos7-luna-demo:
        box: centos7
        memory: 10240
  roles:
    - forklift
```

And then set `- hosts: centos7-luna-demo` on the line below.